patterns/bro

```diff
@@ -0,0 +1,2 @@
+# http.log (https://www.bro.org/sphinx/scripts/base/protocols/http/main.bro.html#type-HTTP::Info)
+HTTP %{NUMBER:ts}\t%{NOTSPACE:uid}\t%{IP:orig_h}\t%{INT:orig_p}\t%{IP:resp_h}\t%{INT:resp_p}\t%{INT:trans_depth}\t%{GREEDYDATA:method}\t%{GREEDYDATA:domain}\t%{GREEDYDATA:uri}\t%{GREEDYDATA:referrer}\t%{GREEDYDATA:user_agent}\t%{NUMBER:request_body_len}\t%{NUMBER:response_body_len}\t%{GREEDYDATA:status_code}\t%{GREEDYDATA:status_msg}\t%{GREEDYDATA:info_code}\t%{GREEDYDATA:info_msg}\t%{GREEDYDATA:filename}\t%{GREEDYDATA:bro_tags}\t%{GREEDYDATA:username}\t%{GREEDYDATA:password}\t%{GREEDYDATA:proxied}\t%{GREEDYDATA:orig_fuids}\t%{GREEDYDATA:orig_mime_types}\t%{GREEDYDATA:resp_fuids}\t%{GREEDYDATA:resp_mime_types}
```

Comment:
Yes the csv filter can be a solution, every bro log must have a own custom "columns" like this:

 __columns => ["ts","uid","id.orig_h","id.orig_p","id.resp_h","id.resp_p",...__

custom separator (\t) and void quote_char. I tested with __quote_char => ""__ and  __quote_char => " "__, but this solution doesn't work.

So I thought to do a custom parser with correct name for every bro logs.